opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -1553,5 +1573,56 @@ mod tests {
                 .downcast_ref::<T>()
                 .expect("Failed to cast aggregation to expected type")
         }
+
+        fn get_from_multiple_aggregations<T: data::Aggregation>(
+            &mut self,
+            counter_name: &str,
+            unit_name: Option<&str>,
+            invocation_count: usize,
+        ) -> Vec<&T> {
+            self.resource_metrics = self
+                .exporter
+                .get_finished_metrics()
+                .expect("metrics expected to be exported");
+
+            assert!(
+                !self.resource_metrics.is_empty(),
+                "no metrics were exported"
+            );
+
+            assert!(
+                self.resource_metrics.len() == invocation_count,
+                "Expected single resource metrics."
```

Comment:
nit: it's probably not the right message? 
```suggestion
                "Expected {} resource metrics and expect {}.",
    self.resource_metrics.len(),
    invocation_count
```

Reply:
Good catch! It was a copy-paste error. I have changed this to use `assert_eq!` instead of `assert`. Let me know what you think.